packages/matrix-identity-server/src/cron/index.test.ts

```
@@ -1,96 +1,86 @@
-                }
-              )
-            })
+              testdb.run("CREATE TABLE IF NOT EXISTS users (name varchar(64) PRIMARY KEY)", (e: unknown) => {
+                if (e !== null) reject(new Error(`Failed to create table users: ${e as string}`));
+                resolve(true);
+              });
+            });
 
-            resolve()
+            resolve();
           } catch (error) {
```

Biome (2.4.9)

[error] This catch binding is unused.

(lint/complexity/noUselessCatchBinding)